packages/venia-concept/src/components/Input/input.css

```diff
@@ -25,29 +17,61 @@
     color: green;
 }
 
+.requiredSymbol {
+    background-color: rgb(var(--venia-text-spot));
+    width: 0.4em;
+    height: 0.4em;
+    margin-top: 0.1em;
+    border-radius: 50%;
+    display: inline-block;
+    vertical-align: top;
+}
+
 .root {
     padding: 0.5rem 1.5rem;
+    --input-right-padding: 1rem;
+}
+
+.rootFocused {
+    composes: root;
+    background-color: rgb(var(--venia-teal-alt));
```

Comment:
Follow: https://github.com/magento-research/pwa-studio/issues/517. Should have a `background-color`, etc.